static/app/views/starfish/components/samplesTable/spanSamplesTable.tsx

```diff
@@ -152,6 +154,11 @@ export function SpanSamplesTable({
       return <ResourceSizeCell bytes={size} />;
     }
 
+    if (column.key === SPAN_DESCRIPTION) {
+      const filename = row[SPAN_DESCRIPTION].split('/').pop().split('?')[0];
+      return <OverflowEllipsisTextContainer>{filename}</OverflowEllipsisTextContainer>;
+    }
```

Comment:
👍🏻 would be great extracted into a function, or even a component like `FilenameCell` or something similar. Up to you, though

Reply:
thanks, added <FilenameCell url={...} >